addons/web/static/tests/fields/relational_fields/field_many2one_tests.js

```diff
@@ -2721,6 +2721,41 @@ QUnit.module('fields', {}, function () {
             form.destroy();
         });
 
+        QUnit.test('do not select a value by pressing TAB on empty many2one', async function (assert) {
+            assert.expect(3);
+
+            const form = await createView({
+                View: FormView,
+                model: 'partner',
+                data: this.data,
+                arch: `<form>
+                        <field name="trululu"/>
+                        <field name="display_name"/>
+                    </form>`,
+            });
+
+            const $input = form.$('.o_field_many2one input');
+            await testUtils.dom.click($input);
+            await testUtils.fields.triggerKeydown($input, 'tab');
+            assert.strictEqual($input.val(), '', "no record should have been selected");
+
+            // open autocomplete dropdown and manually select item by UP/DOWN key and press TAB
+            await testUtils.dom.click($input);
+            await testUtils.fields.triggerKeydown($input, 'down');
+            await testUtils.fields.triggerKeydown($input, 'tab');
+            assert.strictEqual($input.val(), 'second record', "no record should have been selected");
```

Comment:
no record should have been selected ???
Maybe: "second record should have been selected " ?